cmd/podman/pods/create.go

```diff
@@ -36,12 +35,13 @@ var (
   You can then start it at any time with the  podman pod start <pod_id> command. The pod will be created with the initial state 'created'.`
 
 	createCommand = &cobra.Command{
-		Use:               "create [options]",
-		Args:              validate.NoArgs,
+		Use:               "create [options] [name]",
```

Comment:
This must be `[NAME]`, in caps, not lower case. And once you fix it to caps, the system tests will (correctly) complain that:
```
   #/vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
   #|     FAIL: '  podman pod create [options] [NAME]' indicates a maximum of 1 args. I invoked it with more, and expected this exit status
   #| expected: '125'
   #|   actual: '0'
   #\^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
I'm a little upset that my system test did not catch this (the lower-case thing). I will need to investigate.

Reply:
It was also missing that podman volume create did  not have a maximum arg check.

Reply:
Thanks for letting me know about that. Looks like I have a lot of bugs in my help-message checker!